resources/public/api/conf/3.0/scenarios/lossClaimType_amend.md

```diff
@@ -0,0 +1,81 @@
+<p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
+<table>
+    <thead>
+        <tr>
+            <th>Header Value (Gov-Test-Scenario)</th>
+            <th>Scenario</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><p>N/A - DEFAULT</p></td>
+            <td><p>Simulates success response.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>FP_FORWARD</p></td>
+            <td><p>Simulates a Foreign Property carried forward scenario.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>FP_SIDEWAYS</p></td>
+            <td><p>Simulates a Foreign Property carried sideways scenario.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>NONFHL_FORWARD</p></td>
+            <td><p>Simulates a Non-FHL carried forward scenario.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>NONFHL_FORWARD_TO_SIDEWAYS</p></td>
+            <td><p>Simulates a Non-FHL carried forward to sideways scenario.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>NONFHL_SIDEWAYS</p></td>
+            <td><p>Simulates a Non-FHL carried sideways scenario.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>NONFHL_SIDEWAYS_FHL</p></td>
+            <td><p>Simulates a FHL carried sideways scenario.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>SE_FORWARD</p></td>
+            <td><p>Simulates a Self Employment carried forward scenario.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>SE_SIDEWAYS</p></td>
+            <td><p>Simulates a Self Employment carried sideways scenario.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>NO_CHANGE</p></td>
+            <td><p>Simulates the scenario where the relief type has not changed.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>TYPE_OF_CLAIM_INVALID</p></td>
+            <td><p>Simulates a Foreign Property carried forward scenario.</p></td>
+        </tr>
+    </tbody>
+    <tbody>
+        <tr>
+            <td><p>NOT_FOUND</p></td>
+            <td><p>Simulates the scenario where a claim type is invalid for the income source.</p></td>
```

Comment:
The description  for NOT_FOUND should be `Simulates the scenario where no data is found.`